### DISCUSSION TITLE
Working with saves from all users

### DISCUSSION THREAD

**Discussion by GottemHams** [bucanero/apollo-ps3#45] [Q&A] [answered]
In pretty much all of the PS3 homebrew guides it's recommended to create a separate user with its own PSN account just for homebrew stuff, to decrease the risk of problems with your legitimate accounts. So I figured that with Apollo I could e.g. copy saves between users and unlock another user's saves, but it seems that currently Apollo only allows us to interact with the currently logged in user's saves. Perhaps I missed something because the feature list does state it's "multi-user" and "quick access to all the save files on [the internal PS3 HDD]", although it is a bit vague about what exactly it entails.

**Comment by bucanero** [comment, marked as answer]
Yes, Apollo only shows the current user's save-games (HDD). On USB shows everything on `/PS3/SAVEDATA/`

So for that usage scenario, you need to:
- copy the savegame from "user account A" to USB, 
- then login to "user B" and use Apollo to resign the USB save.
- Finally you can copy the savegame to "user B" HDD.

**Comment by GottemHams** [comment]
I don't think that will work for locked saves though, in that case I'd first have to run Apollo with my primary account and that's something I'd like to avoid.

**Comment by bucanero** [comment]
then use FTP to copy anything (locked or not) from one account to the other, and just use Apollo for resigning.

**Comment by GottemHams** [comment]
Strangely enough that doesn't seem to work; I can copy a save's directory but Apollo doesn't see it. I verified that both the original and copied save directories have the same structure and none of the files have become corrupt somehow.

**Comment by bucanero** [comment]
try FTP and copy the folder to your USB drive, inside the `/PS3/SAVEDATA/` path. Apollo should see the USB save files

**Comment by GottemHams** [comment]
That does seem to work, although FTP is pretty slow and when copying multiple directories at once I often lose connection. Would it be possible add a feature to Apollo to allow it to switch between users? Most of your code wouldn't have to change, you'd simply scan a different home directory.

**Comment by GottemHams** [comment]
Why was this closed without any response to my follow-up question? I think it's a legitimate feature and it may be helpful to many others as well. Not to mention the suggested workaround doesn't work reliably, so the issue at hand hardly qualifies as resolved.

**Reply by bucanero** [reply]
❌  your issue/question wasn't closed, it was just moved to the `Discussions` area.

Do you really think that you're entitled to get whatever you request just because "you don't like it"?

But now that you ask again, I'll give my answer: the source code is there, you're free to develop yourself.
